apps/lead/views.py

```diff
@@ -651,57 +761,6 @@ def _get_clone_ready(obj, lead):
             _get_clone_ready(emm_trigger, lead) for emm_trigger in emm_triggers
         ])
 
+        if return_existing_lead:
+            return lead, True
         return lead
```

Comment:
There are two different return types. Might cause issues at the calling end.

Reply:
@bewakes Thanks for the review. Could you check out this commit https://github.com/the-deep/server/pull/476/commits/7d0aca86e85076f59ae6bfdcc31e9f12ae72e9dd, if it resolves?

Reply:
Perfect!